README.md

```diff
@@ -4,36 +4,47 @@
 
 > Web page to Showcase an Interactive To-do List as Microverse Project.
 
+Minimalist To-Do webapp that allows the user to stay organized by:
+
+- Creating a list of tasks
+- Mark task as completed
+- Remove, Edit and Re-order tasks
+- Remove all completed tasks
+
 ![screenshot](src/Snapshot.png)
 
 ## Live Demo
```

Comment:
As mentioned by the previous reviewer we need to add a live demo link.

Reply:
The previous reviewer said, "You could also change it to coming soon if you're managing to add one later" so i changed to coming soon!

Reply:
Deployment is not a must in the project requirements. I've spent the whole night without sleep solving the challenges, so preventing a project to pass due to something that does not affect project functionality and that is not a mandatory requirement is disrespectful and demotivating.

Reply:
Hi @2bleO 
Sorry for the confusion, the live demo is not a hard requirement for this project. However as stated by the first reviewer, since it does not contain the live project content, there is no need for it for now. You can just remove it to maintain the professionalism of your repo and then maybe add it later as advised.

Reply:
thanks @ZawadiSumba66, I appreciate it.